src/platform/backends/libvirt/libvirt_wrapper.cpp

```diff
@@ -44,100 +39,60 @@ auto get_symbol_address_for(const std::string& symbol, void* handle)
     auto address = dlsym(handle, symbol.c_str());
 
     if (!address)
```

Comment:
They shouldn't return null since we are relying on the libvirt API to keep those symbol names.  But if, for whatever reason, they dropped an API function we rely on or renamed it, we need to catch that and be made aware of the change.  This is mainly our purposes to figure out what is happening if `dlsym` can't find the symbol within the loaded `libvirt` .so.  I hope that explains this.  But yes, we never want null symbols.  Otherwise, if we call the function, it will crash.

Reply:
OK, cool.